app/build.gradle

```diff
@@ -111,5 +125,28 @@ dependencies {
     // a debug bridge to inspect Database, Network, Sharedpreferences
     implementation Deps.stetho
     implementation Deps.stethoOkhttp3Interceptor
+
+
+    // AndroidX Test - Instrumented testing
+    androidTestImplementation "androidx.test:core-ktx:${Versions.testCoreKtxVer}"
+    androidTestImplementation "androidx.test.ext:junit-ktx:${Versions.testExtJunitKtxVer}"
+    androidTestImplementation "androidx.test:rules:${Versions.testRulesVer}"
+    androidTestImplementation "androidx.room:room-testing:${Versions.testRoomTestingVer}"
+    androidTestImplementation "androidx.arch.core:core-testing:${Versions.archCoreTestingVer}"
+    androidTestImplementation "androidx.test.espresso:espresso-core:${Versions.testEspressoVer}"
+    androidTestImplementation "androidx.test.espresso:espresso-contrib:${Versions.testEspressoVer}"
+    androidTestImplementation "androidx.test.espresso:espresso-intents:${Versions.testEspressoVer}"
+    androidTestImplementation "androidx.test.espresso.idling:idling-concurrent:${Versions.testEspressoVer}"
+    androidTestImplementation "org.robolectric:annotations:${Versions.robolectricAnnotationsVer}"
+    implementation "androidx.test.espresso:espresso-idling-resource:${Versions.testEspressoVer}"
+
+    androidTestImplementation "androidx.test:runner:${Versions.testRunnerVer}"
+    androidTestImplementation "androidx.test.ext:junit:${Versions.testExtJunitVer}"
+
+    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:${Versions.kotlinxCoroutinesAndroidVer}"
+    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:${Versions.kotlinxCoroutinesTestVer}"
+
+    androidTestImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:${Versions.kotlinxCoroutinesAndroidVer}"
+    androidTestImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:${Versions.kotlinxCoroutinesTestVer}"
```

Comment:
You really need all this dependencies from here to top? You can add only concrete one.